core/options.js

```diff
@@ -213,7 +248,7 @@ Blockly.Options.prototype.getMetrics;
  *    behaviour is unspecified.
  * @param {!Object} options Dictionary of options.
  * @param {boolean} hasCategories Whether the workspace has categories or not.
- * @return {!Object} A dictionary of normalized options.
+ * @return {!Blockly.Options.MoveOptions} Normalized move options.
  * @private
  */
 Blockly.Options.parseMoveOptions = function(options, hasCategories) {
```

Comment:
Slightly unrelated, but should be Blockly.Options.parseMoveOptions_

Reply:
I saw that and was tempted to change, but I imagined there was a good reason to not rename.
I don't imagine developers using this API call, so I'm going to rename in this PR.

Reply:
Marking as api_change just in case.